packages/battery_plus/battery_plus/example/integration_test/battery_plus_test.dart

```diff
@@ -13,11 +13,8 @@ import 'package:integration_test/integration_test.dart';
 void main() {
   IntegrationTestWidgetsFlutterBinding.ensureInitialized();
 
-  final bool batteryLevelIsImplemented = Platform.isAndroid || Platform.isMacOS;
-  final bool batteryStateIsImplemented = Platform.isAndroid ||
-      Platform.isIOS ||
-      Platform.isMacOS ||
-      Platform.isWindows;
```

Comment:
Why of this change?

Reply:
I thought those tests were already running everywhere else. Didn't know that they failed on simulators on the CI. Changed to a working version.